### PR TITLE
move lesson from mod-2, minor updates to example

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,79 @@
...
+    private SteeringWheel steeringWheel;
+    private Engine engine;
+    ...
+}
+```
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+This is a very simplified example, but it shows that we know a car has as
+parts a steering wheel, an engine, etc. So we can represent this in code
```
Comment (kcleland0818) [inline README.md]: "it shows that we know a car has a parts a steering wheel, an engine, etc."

^ needs to be reworded. Suggestion: "it shows that we know a car has a steering wheel, an engine, and possibly other parts."

File: README.md [hunk 2]
```diff
@@ -1,32 +1,79 @@
...
+by defining the `Car` class as having fields to reference 
+`SteeringWheel` and `Engine` objects.
```
Comment (kcleland0818) [inline README.md]: Re-word suggestion: "We can represent this in code by defining a `Car` class to have fields referencing a `SteeringWheel` and an `Engine` object."

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: reworded.

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: reworded.

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**